docs/requirements.txt

```diff
@@ -3,4 +3,4 @@ sphinx==3.2.1
 sphinx_rtd_theme==0.4.3
 sphinxcontrib-programoutput==0.16
 git+https://github.com/pytorch/pytorch_sphinx_theme.git#egg=pytorch_sphinx_theme
-torch==1.6.0
+torch>=1.6.0
```

Comment:
oh, nice catch

Reply:
This file is used by the readthedocs service. The impact of this is that the readthedocs service will now use latest PyTorch after every commit to generate our documentation. That might be OK since we now try to support latest PyTorch ASAP.